Makefile

```diff
@@ -1,28 +1,47 @@
-# Make settings - @see https://tech.davis-hansson.com/p/make/
 SHELL := bash
 .ONESHELL:
 .SHELLFLAGS := -eu -o pipefail -c
 .DELETE_ON_ERROR:
 MAKEFLAGS += --warn-undefined-variables
 MAKEFLAGS += --no-builtin-rules
 
-# Log levels
-DEBUG := $(shell printf "\e[2D\e[35m")
-INFO  := $(shell printf "\e[2D\e[36m🔵 ")
-OK    := $(shell printf "\e[2D\e[32m🟢 ")
-WARN  := $(shell printf "\e[2D\e[33m🟡 ")
-ERROR := $(shell printf "\e[2D\e[31m🔴 ")
-END   := $(shell printf "\e[0m")
+WASM_TARGET=wasm32-wasi
+FASTLY_PACKAGE=fastly-edge-assignments
+BUILD_DIR=target/$(WASM_TARGET)/release
+WASM_FILE=$(BUILD_DIR)/$(FASTLY_PACKAGE).wasm
 
-.PHONY: default
-default: help
-
-## help - Print help message.
+# Help target for easy documentation
 .PHONY: help
-help: Makefile
-	@echo "usage: make <target>"
-	@sed -n 's/^##//p' $<
+help:
+	@echo "Available targets:"
+	@echo "  all                           - Default target (build workspace)"
+	@echo "  workspace-build               - Build the entire workspace excluding the Fastly package"
+	@echo "  workspace-test                - Test the entire workspace excluding the Fastly package"
+	@echo "  fastly-edge-assignments-build - Build only the Fastly package for WASM"
+	@echo "  fastly-edge-assignments-test  - Test only the Fastly package"
+	@echo "  clean                         - Clean all build artifacts"
 
 .PHONY: test
 test: ${testDataDir}
 	npm test
+
+# Build the entire workspace excluding the `fastly-edge-assignments` package
+.PHONY: workspace-build
+workspace-build:
+	cargo build --workspace --exclude $(FASTLY_PACKAGE)
+
+# Run tests for the entire workspace excluding the `fastly-edge-assignments` package
+.PHONY: workspace-test
+workspace-test:
+	cargo test --workspace --exclude $(FASTLY_PACKAGE)
+
+# Build only the `fastly-edge-assignments` package for WASM
+.PHONY: fastly-edge-assignments-build
+fastly-edge-assignments-build:
```

Comment:
Are these changes helpful? I got tired of putting the various parameters in the right place.